test/signup.test.js

```diff
@@ -0,0 +1,59 @@
+const chai = require('chai');
+const app = require('../index');
+const request = require('supertest')(app);
+const db = require('../models/');
+const mocha = require('mocha');
+const faker = require('faker');
+
+const describe = mocha.describe;
+
+const expect = chai.expect; 
+const user =  {
+    email: faker.internet.email(),
+    username:faker.name.lastName(),
+    password: faker.internet.password(),
+};
+
+
+describe('POST requests', function() {
+    beforeEach(function(done){
+      db.User.destroy({
+          where: {},
+      }).then(user => console.log(user));
+      done();
+        
+    });
+     describe('/api/users/', function(){
+
+        describe('status code', function(){
+            it('should be equal to 201', function(done){
```

Comment:
a proper wording would be `should be equal to 201 when a user signs up successfully`

Reply:
Noted